packages/devp2p/test/integration/dpt-simulator.spec.ts

```diff
@@ -8,7 +8,7 @@ describe('DPT simulator tests', () => {
   it('DPT: new working node', () => {
     const dpts = util.initTwoPeerDPTSetup(41622)
 
-    dpts[0].events.on('peer:new', async (peer: any) => {
```

Comment:
It's actually great to see these events typed in a more meaningful way 🙂, have struggled with what "peer" type is being returned when using the API before.